docs/source/tutorial/installation/installation-basics.md

```diff
@@ -6,7 +6,7 @@ JupyterHub is supported on Linux/Unix based systems. To use JupyterHub, you need
 a Unix server (typically Linux) running somewhere that is accessible to your
 team on the network. The JupyterHub server can be on an internal network at your
 organization, or it can run on the public internet (in which case, take care
-with the Hub's [security](./getting-started/security-basics)).
+with the Hub's [security](security-basics).
```

Comment:
There is an open parenthesis just before "in which case", so we need two closing parentheses here :)